src/initializeCacheClient.js

```diff
@@ -4,12 +4,25 @@ import { bind, getContainer, getMetadata } from '@globality/nodule-config';
 
 import Cache from './memcached-promisify';
 
+
+function logCacheEvent(logger, eventType, details) {
+    logger.warning(null, `memcached server event: ${eventType}`, details);
+}
+
 function createCacheClient() {
-    const { config } = getContainer();
+    const { config, logger } = getContainer();
     const memcachedConfig = get(config, 'cache.memcached');
-    return new Cache(
+    const cache = new Cache(
         memcachedConfig,
     );
+    if (logger) {
+        cache.client.on('issue', details => (logCacheEvent(logger, 'issue', details)));
+        cache.client.on('failure', details => (logCacheEvent(null, 'failure', details)));
+        cache.client.on('reconnecting', details => (logCacheEvent(null, 'reconnecting', details)));
+        cache.client.on('reconnect', details => (logCacheEvent(null, 'reconnect', details)));
+        cache.client.on('remove', details => (logCacheEvent(null, 'remove', details)));
```

Comment:
why is 1st argument - logger set to null 🤓  am I missing something here?

Reply:
nope :(